tests/apps_test.go

```diff
@@ -180,8 +181,11 @@ var _ = Describe("deis apps", func() {
 			Specify("that user can retrieve logs for that app", func() {
 				sess, err := cmd.Start("deis logs -a %s", &user, app.Name)
 				Eventually(sess).Should(SatisfyAll(
-					Say("INFO \\[%s\\]: %s created initial release", app.Name, user.Username),
-					Say("INFO \\[%s\\]: domain %s added", app.Name, app.Name)))
+					Say(`(.+) (deis\[controller\]: INFO config test\-.* updated)`),
+					Say(`(.*) (deis\[controller\]: INFO test\-.* created initial release)`),
+					Say(`(.*) (deis\[controller\]: INFO appsettings test\-.* updated)`),
+					Say(`(.*) (deis\[controller\]: INFO domain test\-.* added)`),
+					Say(`(.*) (deis\[controller\]: INFO build test\-.* created)`)))
```

Comment:
Can we add a check for the formatting of app-specific logging, e.g.
```
2016-10-24T19:29:15+00:00 kabuki-fireball[cmd.v2.athvd]: 2016/10/24 19:29:15 10.128.2.21:57488 GET /
```
Perhaps by curling the test app endpoint and then checking logs (minimally for `<appname>[<type>.<release>.<process_id>`)?